.github/workflows/pages.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           docker/doxygen
         docker push ghcr.io/maison-de-la-simulation/ddc/doxygen:${GITHUB_SHA:0:7}
     - name: Publish
-      if: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
+      if: ${{ github.event_name == 'push' && github.ref_name == 'main' && github.repository == 'Maison-de-la-Simulation/ddc' }}
       run: |
         docker push ghcr.io/maison-de-la-simulation/ddc/doxygen
```

Comment:
the first push with the commit sha1 as tag is used in the following steps, the one without tag (hence the `latest` auto tag) represents the top of `main` and is only pushed in case of merge into main